analysis_options.yaml

```diff
@@ -7,7 +7,7 @@
 
 # The following line activates a set of recommended lints for Flutter apps,
 # packages, and plugins designed to encourage good coding practices.
-include: package:flutter_lints/flutter.yaml
+include: package:lints/recommended.yaml
```

Comment:
I would recommend to continue using the `flutter_lints/flutter.yaml` here. That file enables some flutter-specific lint rules and includes all the general rules from `lints/recommended.yaml`.

Reply:
For my own understanding (and to see if we have this documented incorrectly somewhere): What made you change this?

Reply:
It was a mistake on my part, for some reason I thought that `flutter_lints/flutter.yaml` was the same as `package:lints/core.yaml` and that `package:lints/recommended.yaml` was a superset.